src/scripts/main.js

```diff
@@ -1,3 +1,199 @@
 'use strict';
 
-// write your code here
+const cells = [...document.querySelectorAll('.field-cell')];
+const start = document.querySelector('.start');
+const rowsCollection = document.querySelectorAll('.field-row');
+const score = document.querySelector('.game-score');
+const messages = document.querySelectorAll('.message');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
+
+cells.forEach((cell, index) => {
+  cell.dataset.num = `${index + 1}`;
+});
+
+function generateEven() {
+  const num = Math.random();
+
+  return num <= 0.1 ? 4 : 2;
+}
+
+function generateCellNoRepeated() {
+  const randomArr = [];
+  const number1 = Math.ceil(Math.random() * 16);
```

Comment:
move `Math.ceil(Math.random() * 16)` to a separate method - `generateRandomNumber()` or smth like this